plugins/@grouparoo/braze/__tests__/export/export-records.ts

```diff
@@ -212,6 +212,7 @@ describe("braze/exportRecords", () => {
               external_id: email1,
               first_name: "John",
               last_name: "Doe",
+              dob: new Date("1980-12-21"),
```

Comment:
This probably on'y actually be a `Date` object. So let's just test with `"1980-12-21"` to make sure it works.

Reply:
same thing for all the tests